pyproject.toml

```diff
@@ -166,7 +145,7 @@ enable_error_code = "ignore-without-code" # same as in pandas
 allow_untyped_globals = false
 allow_redefinition = false
 local_partial_types = false
-implicit_reexport = false # pyright behaves the same
+implicit_reexport = true
```

Comment:
Could re-enable this option again as mypy 0.971 seems to need `npt = numpy.typing` in either case